### PR TITLE
Remove circomjs, recompile with rust Circom v2, return verified state…

### PR DESCRIPTION
**Pull request by rushby**
Remove circomjs, recompile with rust Circom v2, return verified state directly to avoid issue with state not updating, added more zkVerify wording, removed Thirdweb, removed old contract functionality that was used originally for verification.

### CODE REVIEW

File: pages/api/proof.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
 
   const salt = 50;
   const solution = getSolution(id);
+  console.log("Solution for game id", id, ":", solution);
```
Comment (yurikoinaba) [inline pages/api/proof.ts:40]: Remove this

File: pages/index.tsx [hunk 2]
```diff
@@ -19,13 +19,13 @@ export default function Mastermind() {
...
     <>
       <NextSeo
-        title="zkMastermind"
-        description="An on-chain code-breaker game built with zero-knowledge proofs"
+        title="zkMastermind x zkVerify"
+        description="An on-chain code-breaker game built with zero-knowledge proofs with verification by zkVerify"
         openGraph={{
-          url: "https://zk-mastermind.vercel.app",
-          title: "zkMastermind",
+          url: "https://zkverify.io",
```
Comment (yurikoinaba) [inline pages/index.tsx]: `url: "https://zkmastermind.zkverify.io"`
This should be the canonical url for the page and used for SEO & social media sharing

File: pages/api/proof.ts (hunk 1 above)
Comment (rushby) [reply pages/api/proof.ts:40]: Its server side only

File: pages/index.tsx (hunk 2 above)
Comment (rushby) [reply pages/index.tsx]: Done

**Review by yurikoinaba — APPROVED** [review]

**Pull request merged**